### ISSUE TITLE
No typings for Render.GetFont

### ISSUE THREAD

**Issue by ghost** [marwuint/onetap-ts#6]
The type definition file does not include Render.GetFont, which has a signature of "GetFont(name: string, size: number, is_windows_font: boolean): Font"

**Comment by MaticBabnik** [comment]
Yep, the typings are pretty outdated, but since neither me or @marwuint have an active sub we cant really fix them. @marwuint should probably just archive the repo.
